redesign/app.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="author" content="@dwyl">
+    <meta name="keywords" content="do, what, you, love, dwyl, mvp, time, tracking, help, developers, code, frontend, backend, tooling, tools">
+    <meta name="robots" content="index, follow, noarchive">
+    <meta name="base" content="https://www.dwyl.com">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+    <link rel="stylesheet" href="https://unpkg.com/tachyons@4.7.0/css/tachyons.min.css"/>
+    <link rel="stylesheet" href="css/dwyl.css">
+    <link rel="shortcut icon" href="/img/favicon.ico">
+    <title>dwyl</title>
+  </head>
+  <body>
+    <nav class="w-100 dwyl-bg-dark-gray tc white relative fixed">
+      <input type="checkbox" id="burger" class="absolute top-2 right-1 dn">
+      <label for="burger" class="dn-ns pointer absolute top-1 right-1">
+        <i class="fa fa-bars white dib f2"></i>
+      </label>
+      <ul class="overflow-hidden menu db-ns w-100 list tc pl0 pt3 mv0 f3 fw3 f5-ns">
+        <li class="absolute top-1 static-ns ph4 mh2 fw3 di-ns pt1 pb3 pv3-ns">
+          <a href="/" class="white link">
+            <img src="./img/common/dwyl-heart-only-logo.png" alt="dwyl logo" class="w1-plus"/>
+            dwyl
+          </a>
+        </li>
+        <li class="ph5 pt5 pb2 tl dn-ns"><a href="/" class="white link">Home</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/portfolio" class="white link">Portfolio</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/values" class="white link">Values</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/team" class="white link">Team</a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/app" class="white link">App
+          <span class="dwyl-bg-gold pa1 br2 f7">BETA</span></a></li>
+        <li class="ph5 di-ns pv2 tl pv0-ns"><a href="/blog" class="dwyl-mint link">Blog</a></li>
+        <li class="ph5 di-ns pt2 pb3 tl pv0-ns"><a href="/#contact" class="white link">Contact</a></li>
+      </ul>
+    </nav>
+
+    <section class="dwyl-bg-lightest-gray center pt6-ns pb5-ns">
+
+      <div class="dib-ns w-50-ns dtc v-top dwyl-dark-gray">
+        <div class="w-60-ns center pt4">
+          <h1 class="mt4-ns mb3 pl4-ns pl3 pl0-m f-xl fw9">Time</h1>
+          <h2 class="pl4-ns pv0 mv1 f1 pl3 pl0-m fw9 white text-shadow">Productivity</h2>
+          <h2 class="pl4-ns pv0 mv0 f1 pl3 pl0-m fw9 white text-shadow">Reimagined</h2>
+          <h3 class="pl4-ns pl3 pl0-m ">Track and analyse your daily activities</h3>
```

Comment:
I think this line needs some space on the right side on mobile:
<img src="https://user-images.githubusercontent.com/6057298/30272565-fe445a14-96ec-11e7-87da-37ce1ffef343.png" width="250">